draft-ietf-core-href.md

```diff
@@ -1000,8 +1020,10 @@ value from a data item in the CRI, the presence of any
    1.  If »cri« is not an absolute CRI reference, then fail this
        algorithm.
 
-   2.  Translate the scheme id into a URI scheme name as per
-       {{scheme-map}}; if a scheme id not in this list is being used,
+   2.  Translate the scheme-id into a URI scheme name as per
+       {{scheme-id}} and
+       {{scheme-map}}; if a scheme-id that corresponds to a scheme
+       number not in this list is being used,
        fail this algorithm.
```

Comment:
This covers the case where the scheme section is a scheme-id. Shouldn't this step cover also the case where the scheme section is a scheme-name instead? That would also need to check whether that scheme-name is among those listed in the table above, and, if not, to fail the algorithm.

Reply:
Didn't we want to make the use of the scheme-id mandatory if that exists?
For all entries in the table, scheme-name is not allowed then; so it seems scheme-name is never allow in this procedure.

Reply:
I don't remember considering this detail before but it makes sense, as now also in the latest text.